bitbox-bridge/release/darwin/release.sh

```diff
@@ -8,9 +8,6 @@ CARGO_HOME=/tmp/cargo \
 TARGET_CC=x86_64-apple-darwin14-clang \
 cargo build --target x86_64-apple-darwin --release
 
-NAME=BitBoxBridge
-VERSION=$(toml-echo bitbox-bridge/Cargo.toml package.version)
-
 (
```

Comment:
It starts a subshell, I usually use that when I switch working directory so that it switches back automatically. Other idioms are `pushd` / `popd` or `cd -`.

Reply:
Yeah, sorry for a rhetorical question. I meant there's no need to preserve work dir here. But it's ok. I'd like to change the scripts to release on native platforms anyway.